install.sh

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 
 ./build.sh
-echo Y | cf install-plugin pcc
+echo Y | cf install-plugin gemfire
 echo "    _______  ________    ________  _______  _______"
 echo "   / _____/ / ______/   / ____  / / _____/ / _____/"
```

Comment:
these "echo" lines print CF PCC in giant letters

Reply:
Also note the hard coded "1.0.0" in this echo. Would it be possible to read these numbers at runtime from VersionType you updated on structs.go?

Reply:
Yeah, makes sense to get them from there.